### DISCUSSION TITLE
How to set Joint limits when creating a multi body object via createMultiBody?

### DISCUSSION THREAD

**Discussion by KarlJosef** [bulletphysics/bullet3#3587] [General]
I am trying to create an robotic hand via the `createMultiBody` functionality. Most of my joints are `JOINT_REVOLUTE` joints. I would like to specify the lower and upper limit of the angle like one can do this in an URDF file via `<limit  lower="-0.17" upper="1.8" />`.  
Is this possible via the current `pybullet` functionality?

**Comment by bdelhaisse** [comment]
It would be nice to have a `setJointInfo()` method in pybullet.
